action.yml

```diff
@@ -304,7 +320,7 @@ runs:
           # We often expect a merge-conflict when there are client/dist file differences
           MERGE_RESULT=$(git merge --no-ff --no-commit $FROM_BRANCH || true)
 
-          # Only merge conflicts in client/dist are allowed, stop for all others
+          # Only merge conflicts in client/dist are allowed, stop for all others, including yarn.lock
           # See https://git-scm.com/docs/git-status#_output for information on the porcelain format
           UNMERGED_FILES=$(git status --porcelain=v1 | grep -P '^(DD|AU|UD|UA|DU|AA|UU)' | grep -v client/dist) || true
           if [[ $UNMERGED_FILES != "" ]]; then
```

Comment:
won't this still halt for `yarn.lock` for merge-ups between minor branches in the same major release series? Or am I missing something here?

Reply:
no, because there shouldn't be a merge-conflict e.g. upgrade minor deps on 6.1, merge up 6.1 to 6, there shouldn't be a merge conflict at this point

If on the other hand you upgrade minor deps on 6.1, and on 6, then merge-up 6.1 to 6, there could be merge-conflicts at that point, in which case we should stop

Reply:
Okay. So it _will_ halt if there's a merge conflict, but because we don't expect conflicts here, the halt is desirable?
That sounds fine to me.